exec/jdbc-all/pom.xml

```diff
@@ -441,7 +441,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
                   
                   </message>
-                  <maxsize>20000000</maxsize>
+                  <maxsize>21000000</maxsize>
```

Comment:
Is this really needed? Looks as though you tried to add a new dependency but failed.